cadc-vos/src/main/java/ca/nrc/cadc/vos/client/VOSpaceClient.java

```diff
@@ -364,6 +383,11 @@ public Node setNode(Node node)
         {
             throw new IllegalStateException("Node not found", e);
         }
+        catch (ResourceAlreadyExistsException e)
+        {
+            log.debug("failed to set node", e);
+            throw new IllegalStateException("failed to set node", e);
```

Comment:
I'm sure there's a good reason why these all end up as IllegalStateExceptions?  I just don't remember...

Reply:
As discussed in Slack - I don't know why, just see that it's used throughout cadc-vos and cadc-vos-server.

Reply:
Believe the decision is to leave this as-is.